tools/multiqc.wdl

```diff
@@ -9,47 +9,58 @@ task multiqc {
     input {
         File sorted_bam
         String validate_sam_string
-        Array[File] qualimap_bamqc
-        Array[File] qualimap_rnaseq
+        File qualimap_bamqc
+        File? qualimap_rnaseq
         Array[File] fastqc_files
+        Array[File] fastq_screen
         File flagstat_file
         File bigwig_file
-        File star_log
+        File? star_log
         Int max_retries = 1
+        String wait_var = ""
     }
 
     Float star_size = size(sorted_bam, "GiB")
     Int disk_size = ceil((star_size * 4) + 10)
 
     command {
-        echo ${sorted_bam} > file_list.txt
-        echo ${validate_sam_string} > validate_sam.txt
+        echo ~{sorted_bam} > file_list.txt
+        echo ~{validate_sam_string} > validate_sam.txt
         echo validate_sam.txt >> file_list.txt
 
-        for file in ${sep=' ' qualimap_bamqc} ; do
+        mkdir qualimap_bamqc/
+        tar -xzf ~{qualimap_bamqc} --strip-components=1 -C qualimap_bamqc/;
```

Comment:
I think this still will present problems in the multiqc report where it shows as two line items in the summary table.

Reply:
<img width="955" alt="Screen Shot 2020-03-12 at 11 18 52 AM" src="https://user-images.githubusercontent.com/28660237/76542875-ab773180-6453-11ea-8c48-4d0128a11f52.png">
You're right. I'm looking at the MultiQC docs to figure out the right way to do this.

Reply:
Do you have any idea why it fails to get the sample name from `bamqc` but succeeds for `qualimap rnaseq`?

Reply:
I wrote a solution, but I don't love it:
```
# MultiQC should be able to strip any extensions on the BAM name
# down to the sample name, and will use that sample name for results
qualimap_bamqc_dir=$(basename ~{sorted_bam} '.bam')
mkdir "$qualimap_bamqc_dir"
tar -xzf ~{qualimap_bamqc} --strip-components=1 -C "$qualimap_bamqc_dir"/;
echo "$qualimap_bamqc_dir"/genome_results.txt >> file_list.txt
for file in `find "$qualimap_bamqc_dir"/raw_data_qualimapReport/`; do
    echo $file >> file_list.txt
done
```
I left `qualimap_rnaseq` alone, because MultiQC could parse the sample name from the file contents.